### PR TITLE
Auto generated help message

### PR DESCRIPTION
**Pull request by Pranjal-Patel**

### CODE REVIEW

File: Swirl/include/cli/cli.h [hunk 1]
```diff
@@ -16,31 +18,27 @@ namespace cli {
...
+    * This function returns the value of the flag requested from the provided args vector.
+    * If the flag is not supplied, it returns false.
+    * If the flag is supplied, it returns the value of the flag.
+    * If the flag is not required to have a value, it returns true.
+    *
+    * @param flag Requested flag
+    * @param args Vector of arguments
+    * @return The value of the flag
+    */
+    std::variant</* The flag value */ std::string, /* The flag is supplied or not */ bool>
```
Comment (MrinmoyHaloi) [inline Swirl/include/cli/cli.h:41]:
```suggestion
    std::variant<std::string, bool>
```
I don't think if those comments are needed. There is already a docstring.

File: Swirl/src/cli/cli.cpp [hunk 2]
```diff
@@ -33,8 +52,8 @@ namespace cli {
 
 		return supplied;
 	}
-    
-    std::variant<std::string, bool>
+
+    std::variant</* The flag value */ std::string, /* The flag is supplied or not */ bool>
```
Comment (MrinmoyHaloi) [inline Swirl/src/cli/cli.cpp:56]: Also here these aren't needed

**Review by MrinmoyHaloi — CHANGES_REQUESTED** [review]

File: Swirl/src/cli/cli.cpp (hunk 2 above)
Comment (Pranjal-Patel) [reply Swirl/src/cli/cli.cpp:56]: removed

**Comment by Pranjal-Patel** [comment]
Ok, i removed those

**Review by MrinmoyHaloi — APPROVED** [review]

**Pull request merged**